test/e2e/tests/test_elasticsearch_domain.py

```diff
@@ -88,7 +95,7 @@ def wait_for_delete_or_die(es_client, resource, timeout):
 @service_marker
 @pytest.mark.canary
 class TestDomain:
-    def test_create_delete_7_9(self, es_client):
+    def test_create_delete_7_9(self, es_client, resources):
```

Comment:
where are you using the `resources` fixture in this method?

Reply:
will update